docqueries/driving_distance/pgr_drivingDistance.test.sql

```diff
@@ -0,0 +1,15 @@
+-- CopyRight(c) pgRouting developers
+-- Creative Commons Attribution-Share Alike 3.0 License : https://creativecommons.org/licenses/by-sa/3.0/
+/* --q5 */
+SELECT * FROM pgr_drivingDistance(
+  'SELECT id, source, target, cost, reverse_cost FROM edges',
+  11, 3.0);
+/* --q6 */
+SELECT * FROM pgr_drivingDistance(
+  'SELECT id, source, target, cost, reverse_cost FROM edges',
+  array[11, 16], 3.0, equicost => true);
```

Comment:
Use capital `ARRAY` at least for newer code (we can also fix this later for older code):
```suggestion
  ARRAY[11, 16], 3.0, equicost => true);
```